kernel/src/actions/visitors.rs

```diff
@@ -277,9 +331,11 @@ mod tests {
 
     use super::*;
     use crate::{
-        actions::{get_log_schema, ADD_NAME},
-        client::arrow_data::ArrowEngineData,
-        client::sync::{json::SyncJsonHandler, SyncEngineInterface},
+        actions::{get_log_schema, ADD_NAME, TRANSACTION_NAME},
+        client::{
+            arrow_data::ArrowEngineData,
+            sync::{json::SyncJsonHandler, SyncEngineInterface},
+        },
```

Comment:
tiny nit: the deep nesting of imports can make it both bigger and harder to read:
```suggestion
        client::arrow_data::ArrowEngineData,
        client::sync::{json::SyncJsonHandler, SyncEngineInterface},
```
(a bit counter-intuitive)